examples/hybrid/TurbulenceConvectionUtils.jl

```diff
@@ -0,0 +1,283 @@
+module TurbulenceConvectionUtils
+
+using LinearAlgebra, StaticArrays
+import ClimaCore
+const CC = ClimaCore
+const CCG = CC.Geometry
+const CCO = CC.Operators
+import ClimaCore.Geometry: ⊗
+
+import UnPack
+
+import OrdinaryDiffEq
+const ODE = OrdinaryDiffEq
+
+import CLIMAParameters
+const CP = CLIMAParameters
+
+import Logging
+import TerminalLoggers
+Logging.global_logger(TerminalLoggers.TerminalLogger())
+
+import TurbulenceConvection
+const TC = TurbulenceConvection
+const tc_dir = pkgdir(TC)
+
+Base.:-(a::FT, b::CCG.Covariant3Vector{FT}) where {FT} = a .- b.u₃
+Base.:+(a::FT, b::CCG.Covariant3Vector{FT}) where {FT} = a .+ b.u₃
```

Comment:
@simonbyrne, these operations weren't previously defined, is this okay in the context of a sphere? The results seem to be fine with a rectangle mesh.

Reply:
It would probably be less confusing to just wrap `a` in a `Covariant3Vector` before calling `+` or `-`.

Reply:
I'll open an issue for this, and remove the base overloading later. I don't think it's too confusing on the EDMF side: the covariant3 vector only has one component, so we basically treat everything like scalars.

Reply:
I've migrated this to a TC issue.